flights/berks_ceh_clients_flight.yaml

```diff
@@ -1,4 +1,4 @@
-# select c.*, e."LastGradeCompleted", e."EmploymentType", e."SchoolStatus", e."Employed", e."EmploymentEducationID", en."EnrollmentID", en."FormerWardJuvenileJustice", en."SexualOrientation" from client as c left join employmenteducation as e on c."PersonalID"=e."PersonalID" left join enrollment as en on e."EnrollmentID"=en."EnrollmentID" AND c."PersonalID"=en."PersonalID" limit 5000;
+# select c.*, e."LastGradeCompleted", e."EmploymentType", e."SchoolStatus", e."Employed", e."EmploymentEducationID", en."EnrollmentID", en."FormerWardJuvenileJustice", en."SexualOrientation", h."DomesticViolenceVictim", h."WhenOccurred", h."CurrentlyFleeing" from client as c left join employmenteducation as e on c."PersonalID"=e."PersonalID" left join enrollment as en on e."EnrollmentID"=en."EnrollmentID" AND c."PersonalID"=en."PersonalID" left join health_and_dv as h on en."EnrollmentID"=h."EnrollmentID" AND c."PersonalID"=h."PersonalID" limit 5000;
```

Comment:
remember to take out the limit 5000 when running it on prod - I assume that was to test it locally?

Reply:
yep!